ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/KsqlResourceFunctionalTest.java

```diff
@@ -126,25 +123,6 @@ public void shouldHandleInterDependantExecutableAndNonExecutableStatements() {
     ));
   }
 
-  @Test
-  public void shouldHandleInterDependantCsasTerminateAndDrop() {
```

Comment:
Feels like an important test you're removing...

Reply:
The main issue with this test is that it's very hard to determine what offset this will be called with since the same command topic is being reused between tests. If I call this test alone vs when I call it with the others will change which offset the query statement is executed as. If you feel this test is very important I could move this to its own separate test file by itself.

Reply:
You could get the current end offset from the local broker, and use that to determine the query ID to expect.

Reply:
Test has been reworked to work with the new id generation.